test/unit/multipart-test.ts

```diff
@@ -30,11 +30,16 @@ type Part = {
     body: string;
 };
 
-function testStreamMixed(body1?: string, body2?: string) {
+interface ITestStreamData {
+    reader: Reader<Buffer>;
+    expectedResult: string;
+}
+
+function testStreamMixed(body1?: string, body2?: string): ITestStreamData {
     const parts = [
         {
             headers: {
-                A: 'VA1',
+                A: 'VÀ1',
```

Comment:
accent is not very visible. What about `Và1`

Reply:
done